src/Blocks/utils.jl

```diff
@@ -0,0 +1,47 @@
+@connector function RealInput(;name, nin=1, u_start=nin > 1 ? 0.0 : zeros(nin))
```

Comment:
I added `RealInput` and `RealOutput` as connectors, such that the components can be connected via `connect`. However, the connectors only have one variable, which makes accessing them a bit redundant. Maybe a this can be handled through MTK via a `connect` for `Num` and just using a plain variable. For the blocks there is no need for a sophisticated stuff like inside and outside connectors or balance equations.